Tests/Acceptance/DoctrineClientManagerTest.php

```diff
@@ -0,0 +1,115 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Trikoder\Bundle\OAuth2Bundle\Tests\Acceptance;
+
+use DateTime;
+use Doctrine\ORM\EntityManagerInterface;
+use Trikoder\Bundle\OAuth2Bundle\Manager\Doctrine\ClientManager as DoctrineClientManager;
+use Trikoder\Bundle\OAuth2Bundle\Model\AccessToken;
+use Trikoder\Bundle\OAuth2Bundle\Model\Client;
+use Trikoder\Bundle\OAuth2Bundle\Model\RefreshToken;
+
+/**
+ * @covers \Trikoder\Bundle\OAuth2Bundle\Manager\Doctrine\ClientManager
```

Comment:
@Allypost I think the same TODO applies here as in DoctrineAccessTokenManagerTest & DoctrineRefreshTokenManagerTest, could you please add it.

Reply:
@HypeMC Fixed. Also added `@covers` tags to the aforementioned classes.